app/controllers/patients_controller.rb

```diff
@@ -324,6 +324,19 @@ def clear_assessment
     history.save
   end
 
+  # A patient is eligible to be removed from a household if their responder doesn't have the same contact
+  # information of them
+  def household_removeable
+    redirect_to(root_url) && return unless current_user.can_edit_patient?
+
+    patient = current_user.get_patient(params.permit(:id)[:id])
+    duplicate_contact = false
+    duplicate_contact = patient[:primary_telephone] == patient.responder[:primary_telephone] unless patient[:primary_telephone]&.blank?
+    duplicate_contact ||= (patient[:email] == patient.responder[:email]) unless patient[:email]&.blank?
```

Comment:
Is it worth including the preferred contact method in this logic? I'm thinking about that case if the preferred contact method was email, and this patient had a unique email, but still had the same phone number as the HOH.

Reply:
I thought about that too and even originally had that logic in there. Only issue with doing it that way is that uses could then switch the primary contact method after-the-fact without re-adding them as a member of the household and then we would see the same duplicate contact error that we see now.

Reply:
This goes back to the discussion of whether it makes sense to have households and responders be two different, independent variables. Lets chat about this today.

Reply:
For that first case, I don't see how we can avoid that without enforcing that on any updates (out of the scope of this PR). The most we can do is require a fix upfront, before allowing to remove from household.

Reply:
I still don't see how the two are separate. My thinking is all this check should do is require the minimal amount of change to the record to ensure no issues come up when reporting. In fact, we probably don't even need to check for email here, because that flow would still work fine.